letsdrill.rb

```diff
@@ -1,13 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
 
+  #Put your code here!
+  case integer
+  when (60..69)
+  	grade = "D"
+  when (70..79)
+  	grade = "C"
+  when (80..89)
+  	grade = "B"
+  when (90 ..100)
+  	grade = "A"
+  else 
+  	grade = "F"
+  end
+  grade
 end
```

Comment:
Overall LGTM, just a few thoughts. The case block already passes out a value, so if you want, you can omit declaring grade at line 16 and this function will work the same. Another thing is a matter of opinion but since these when statements have such small bodies, you might consider making them one-liners through the use of the `when ... then ...` syntax.